client/src/main/scala/drt/client/components/TerminalDesksAndQueuesRow.scala

```diff
@@ -57,20 +59,32 @@ object TerminalDesksAndQueuesRow {
                 case pc if pc >= 0.7 => "amber"
                 case _ => ""
               }
-              List(paxLoadTd,
-                <.td(^.className := queueColour(qn), ^.title := s"Rec: ${cm.deskRec}", s"${cm.deployedDesks.getOrElse("-")}"),
-                <.td(^.className := s"${queueColour(qn)} $ragClass", ^.title := s"With rec: ${cm.waitTime}", s"${cm.deployedWait.map(Math.round(_)).getOrElse("-")}"))
+              if(props.showWaitColumn){
+                List(paxLoadTd,
+                  <.td(^.className := s"${queueColour(qn)} $ragClass", ^.title := s"Rec: ${cm.deskRec}", s"${cm.deployedDesks.getOrElse("-")}"),
+                  <.td(^.className := s"${queueColour(qn)} $ragClass", ^.title := s"With rec: ${cm.waitTime}", s"${cm.deployedWait.map(Math.round(_)).getOrElse("-")}"))
+              }else {
+                List(paxLoadTd, <.td(^.className := s"${queueColour(qn)} $ragClass", ^.title := s"Rec: ${cm.deskRec}", s"${cm.deployedDesks.getOrElse("-")}"))
+              }
             case ViewRecs =>
               val ragClass = slaRagStatus(cm.waitTime.toDouble, props.airportConfig.slaByQueue(qn))
-              List(paxLoadTd,
-                <.td(^.className := queueColour(qn), ^.title := s"Dep: ${cm.deployedDesks.getOrElse("-")}", s"${cm.deskRec}"),
-                <.td(^.className := s"${queueColour(qn)} $ragClass", ^.title := s"With Dep: ${cm.waitTime}", s"${Math.round(cm.waitTime)}"))
+              if(props.showWaitColumn) {
+                List(paxLoadTd,
+                  <.td(^.className := s"${queueColour(qn)} $ragClass", ^.title := s"Dep: ${cm.deployedDesks.getOrElse("-")}", s"${cm.deskRec}"),
+                  <.td(^.className := s"${queueColour(qn)} $ragClass", ^.title := s"With Dep: ${cm.waitTime}", s"${Math.round(cm.waitTime)}"))
+              } else {
+                List(paxLoadTd, <.td(^.className := s"${queueColour(qn)} $ragClass", ^.title := s"Dep: ${cm.deployedDesks.getOrElse("-")}", s"${cm.deskRec}"))
```

Comment:
Same here: We can move this repeated `td` into its own function to remove the repetition